lti_consumer/lti_consumer.py

```diff
@@ -558,6 +617,23 @@ def role(self):
         """
         return ROLE_MAP.get(self.runtime.get_user_role(), u'Student')
 
+    @property
+    def user_roles(self):
+        """
+        Get system user role and convert it to LTI 1.3 user roles.
+        """
+        roles = ['student']
+
+        if self.runtime.get_user_role() == 'instructor':
+            # Instructor have both instructor and administrator
+            # permissions on LTI tool.
```

Comment:
Is this still the case? I thought we were aiming for a 1:1 mapping of roles.

Reply:
I don't think a 1:1 role mapping is a good solution here.

If we do so, course creators (`staff`) won't have admin access to a LTI Tool and won't be able to manage users/moderate content inside LTI. This is the same reasoning why the LTI 1.1 implementation does the mapping like this.